PKHeX.WinForms/Subforms/PKM Editors/MemoryAmie.cs

```diff
@@ -435,10 +435,10 @@ private void changeCountryText(object sender, EventArgs e)
 
         private void update255_MTB(object sender, EventArgs e)
         {
-            MaskedTextBox mtb = sender as MaskedTextBox;
+            MaskedTextBox mtb = (MaskedTextBox) sender;
```

Comment:
Type conversion is slower because it needs to consider user-defined conversion code if applicable, while type casting simply checks whether or not an object is a certain type.

Here's a newer article:
http://theburningmonk.com/2010/04/net-tips-using-as-vs-casting-in-csharp/